src/tests/Common/android-test-runner/AndroidTestRunner.proj

```diff
@@ -0,0 +1,46 @@
+<Project Sdk="Microsoft.Build.NoTargets/1.0.53" DefaultTargets="BuildApp">
+  <PropertyGroup>
+    <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
+    <MicrosoftNetCoreAppRuntimePackDir>$(CORE_ROOT)/runtimepack</MicrosoftNetCoreAppRuntimePackDir>
+    <AndroidAppBuilderDir>$(CORE_ROOT)/AndroidAppBuilder</AndroidAppBuilderDir>
+    <EnableTargetingPackDownload>false</EnableTargetingPackDownload>
+  </PropertyGroup>
+
+  <UsingTask TaskName="AndroidAppBuilderTask"
+             AssemblyFile="$(AndroidAppBuilderDir)\AndroidAppBuilder.dll" />
+
+  <!-- Build APK during 'dotnet publish' -->
+  <Target Name="BuildAppBundle" AfterTargets="Build">
+    <PropertyGroup>
+      <AndroidAbi Condition="'$(Platform)'=='arm64'">arm64-v8a</AndroidAbi>
```

Comment:
What architectures do we intend to support on Android? This bit refers to arm64 / arm / x64, in another place we're checking for "android_x64 & android_x86" and in yet another place we hardcode the RID to be "android_x64". It might be useful to be more explicit about "what android architectures make sense or are planned to be ultimately supported" vs. "what android architectures are currently supported" and ideally formulate most of the script provisions for the general case even though right now you're just standing up a single targeting architecture.

Reply:
We plan to support all of arm64, arm and x64. I will update all places to use arch variable.

Reply:
This PR is to support x64 only.